### DISCUSSION TITLE
Export for Frictionless Data Packages

### DISCUSSION THREAD

**Discussion by jze** [directus/directus#16730] [Feature Requests]
It would be nice if data could also be exported as [Frictionless Data](https://frictionlessdata.io/) Packages.

Because we have precise information about the fields in Directus, it would be possible to create meaningful package descriptions.

Exporting a single collection as a [Tabular Data Resource](https://specs.frictionlessdata.io/tabular-data-resource/) would also be useful.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
